lib/plugins/blocks.js

```diff
@@ -110,19 +110,21 @@ function inject (bot, { version, storageBuilder }) {
     bot.emit('chunkColumnLoad', columnCorner)
   }
 
-  function waitForChunksToLoad (cb) {
+  async function waitForChunksToLoad () {
     // check 5x5 chunks around us
     for (let x = -2; x <= 2; x++) {
       for (let y = -2; y <= 2; y++) {
         if (bot.world.getColumnAt(bot.entity.position.plus(new Vec3(x, 0, y).scaled(16))) == null) {
           // keep wait
-          return setTimeout(() => {
-            waitForChunksToLoad(cb)
-          }, 100)
+          return new Promise((resolve) =>
+            setTimeout(async () => {
+              await waitForChunksToLoad()
+              resolve()
+            }, 100)
+          )
```

Comment:
```suggestion
         await new Promise(resolve => setTimeout(resolve, 100))
         await waitForChunksToLoad()
```

Reply:
Semantically it's the same of course but I agree it looks cleaner. Keep in mind (in general) that I tried to change semantics as little as possible in the beginning, stuff like this can also easily be done later. But I agree in this case this is a good improvement